### ISSUE TITLE
Total War: Shogun 2 (34330)

### ISSUE THREAD

**Issue by Patola** [ValveSoftware/Proton#7110]
# Compatibility Report
- Name of the game with compatibility issues: Total War: Shogun 2
- Steam AppID of the game: 34330
* Note: there is an older compatibility report of Shogun 2 but it's closed.

## System Information
- GPU: RX 7900 XTX
- Video driver version: Mesa 23.1.7
- Kernel version: 6.1.53-1-lts
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/Patola/93face5f67aff4041cb0105463ff6d42
- Proton version: 8.0-3. Also tried 7.0-6

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.

[steam-34330.log](https://github.com/ValveSoftware/Proton/files/12644310/steam-34330.log)
This log had DirectX11 selected and as soon as the black screen crashed the "play" button turned green again.

## Symptoms 
- Game crashes

## Reproduction
- Install SHOGUN 2 with Proton 8.0-3. Once downloaded, press play. It will show you the launcher, press play, select DirectX 11. The screen blacks out for a few seconds, the cursor changes to a knife for a moment, then changes back to a regular cursor and you are back to the desktop but there is no game window and often the game is still running (you have to press "stop").
- If you run the game but choose DirectX 9, it works correctly but when closing still stays awake until you press "stop".
- This entry is specially important because the linux native port of the game stopped working due to glibc changes and will probably never work again, so users are depending on proton to play it.

**Comment by kisak-valve** [comment]
Off-topic for Proton support, but it's worth noting that the Linux native variant of this game is probably affected by https://github.com/ValveSoftware/steam-runtime/issues/613.

**Comment by kisak-valve** [comment]
`wine: Call from 7B04F057 to unimplemented function d3dx11_42.dll.D3DX11LoadTextureFromTexture, aborting` looks like the line of interest from your Proton log.

**Comment by Patola** [comment]
> `wine: Call from 7B04F057 to unimplemented function d3dx11_42.dll.D3DX11LoadTextureFromTexture, aborting` looks like the line of interest from your Proton log.

Indeed, doing a `protontricks 34330 d3dx11_42` fixed the issue. Also submitted a protondb report so that other people know what to do.

**Comment by YellowOnion** [comment]
@kisak-valve Sadly all of Creative assembly's Linux games are not compatible with Windows multiplayer due to using different libraries, So proton-support is basically a must if I want to play with my friends.

For some reason I cannot get winetricks to setup d3dx11_42 correctly, and my distro's protontricks is extremely out of date.
